sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -797,11 +794,11 @@ ProgramManager::ProgramPtr ProgramManager::build(
     LinkOpts = LinkOptions.c_str();
   }
 
-  // TODO: Because online linking isn't implemented yet on Level Zero, the
-  // compiler always links against the fallback device libraries.  Once
-  // online linking is supported on all backends, we should remove the line
-  // below and also change the compiler, so it no longer links the fallback
-  // code unconditionally.
+  // TODO: Currently, online linking isn't implemented yet on Level Zero.
```

Comment:
Hi, @kbobrovs 
The "TODO" has been added here, we will enable online spv link again once Level Zero supports.
Thank you very much.